RevenueCatUI/UIKit/PaywallFooterViewController.swift

```diff
@@ -51,6 +51,25 @@ public final class PaywallFooterViewController: PaywallViewController {
     public required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    public override func loadView() {
+        super.loadView()
+
+        self.addChild(self.hostingController)
+        self.view.addSubview(self.hostingController.view)
+
+        NSLayoutConstraint.activate([
```

Comment:
Hmm won't the parent's layout mess with this?

Reply:
I would change the parent implementation to use constraints, expose the top constraint, and deactivate it in this subclass.

Reply:
it doesn't in my testing, but that sounds like a good approach